docs/en/manuals/live-update.md

```diff
@@ -127,218 +61,46 @@ When bundling, any excluded resource will be left out of the application bundle.
+One way to do so is to add an extra file to the zip archive after the game has been bundled. For instance by inserting a `metadata.json` with any relevant information that the game requires. Then, at startup, the game can retrieve with `sys.load_resource("/metadata.json")`. _Note that you will need a unique name for each mount's custom data, or the mounts will give you the file with the topmost priority_
 
-To use the Defold Live update feature together with Amazon services you need an Amazon Web Services account. If you don't already have an account you can create one here https://aws.amazon.com/.
+Failure to do so, you may end up in a situation where the content is not compatible with the engine at all, forcing it to quit.
 
-This section will explain how to create a new user with limited access on Amazon Web Services that can be used together with the Defold editor to automatically upload Live update resources when you bundle your game, as well as how to configure Amazon S3 to allow game clients to retrieve resources. For additional information about how you can configure Amazon S3, please see the [Amazon S3 documentation](http://docs.aws.amazon.com/AmazonS3/latest/dev/Welcome.html).
+## Mounts
 
-1. Create a bucket for Live update resources
+The live update system can use multiple content archives at the same time.
+Each archive is "mounted" to the engine's resource system, with a name and priority.
 
-    Open up the `Services` menu and select `S3` which is located under the _Storage_ category ([Amazon S3 Console](https://console.aws.amazon.com/s3)). You will see all your existing buckets together with the option to create a new bucket. Though it is possible to use an existing bucket, we recommend that you create a new bucket for Live update resources so that you can easily restrict access.
+If two archives have the same file `sprite.texturec`, the engine will load the file from the mount with the highest priority.
 
-    ![Create a bucket](images/live-update/01-create-bucket.png)
+The engine doesn't keep a reference to any resource in a mount. Once a resource is loaded into memory, the archive may be unmounted. The resource will remain in memory until it is unloaded.
 
-2. Add a bucket policy to your bucket
+The mounts are automatically readded upon engine restart.
 
-    Select the bucket you wish to use, open the *Properties* panel and expand the *Permissions* option within the panel. Open up the bucket policy by clicking on the *Add bucket policy* button. The bucket policy in this example will allow an anonymous user to retrieve files from the bucket, which will allow a game client to download the Live update resources that are required by the game. For additional information about bucket policies, please see [the Amazon documentation](https://docs.aws.amazon.com/AmazonS3/latest/dev/using-iam-policies.html).
+_NOTE:_ Mounting an archive doesn't copy or move the archive. The engine only stores the path to the archive.
```

Comment:
::: sidenote
Mounting an archive doesn't copy or move the archive. The engine only stores the path to the archive.
:::